src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -149,6 +158,8 @@
     private boolean restCodeReview;
     private boolean restVerified;
     private boolean gerritBuildCurrentPatchesOnly;
```

Comment:
A slightly better approach, to slightly lesser the amount of fields in this class would be to deprecate ```gerritBuildCurrentPatchesOnly```, make it transient and replace it with an object containing ```AbortNewPatchsets``` and ```AbortManualPatchsets```. It would also make the optionalBlock _(commented below)_ simpler to implement I believe.